ADAL/src/request/ADAcquireTokenSilentHandler.m

```diff
@@ -126,6 +127,14 @@ - (void)acquireTokenByRefreshToken:(NSString*)refreshToken
         request_data[OAUTH2_SCOPE] = _requestParams.scope;
     }
     
+    if (![ADHelpers isADFSInstance:_requestParams.authority])
+    {
+        NSString* enrollId = [ADEnrollmentGateway enrollmentIDForUniqueAccountID:nil userID:(cacheItem.userInformation.userId ? cacheItem.userInformation.userId : _requestParams.identifier.userId) error:NULL];
```

Comment:
very nit: this conditional is a bit long to read in a combination with the method call :)